lib/xapi-stdext-unix/unixext.mli

```diff
@@ -67,10 +67,12 @@ val string_of_file : string -> string
     the operation the file with the path [fname] is not modified at all. *)
 val atomic_write_to_file : string -> Unix.file_perm -> (Unix.file_descr -> 'a) -> 'a
 
-(** Atomically write a string to a file *)
+(** [write_string_to_file] [fname] [contents] creates a file with path [fname]
```

Comment:
I think the usual way to write it is to put it all in a single `[]`, like `[write_string_to_file fname contents]`.